api_data.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import json
+import sys
+
+"""Py version check"""
+if sys.version_info[0] == 3:
+    from urllib.request import urlopen
+else: 
+	from urllib import urlopen 
+
+def output_cve_id(type=None, name=None, version=None): 
+	"""Take as input, a package name, package version. 
+	Queries cve-search' dataset for any reported 
+	vulnerabilities of the requested package. If 
+	vulnerability exists, outputs cve-id(s).
+	"""
+	if version:
```

Comment:
@pombredanne As mentioned above, i wasn't sure if going with, keeping the function in a try-except block an effective way to counter this?